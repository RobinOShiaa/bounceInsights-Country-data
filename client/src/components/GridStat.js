//a gridbox stating a field and q value of retrospective data
export const GridStat = ({ property, value }) => {
  return (
    <div className="bg-gray-900 px-4 py-6 sm:px-6 lg:px-8">
      <p className="text-sm font-medium leading-6 text-gray-400">{property}</p>
      <p className="mt-2 flex items-baseline gap-x-2">
        <span className="truncate text-xl font-semibold tracking-tight text-white">
          {value}
        </span>
      </p>
    </div>
  );
};
