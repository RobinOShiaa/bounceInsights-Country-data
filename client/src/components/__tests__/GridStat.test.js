import { GridStat } from "../GridStat";
import { render } from "@testing-library/react";

test("Matches GridStat component snapshot", () => {
  const tree = render(<GridStat property="testProperty" value="testValue" />);
  expect(tree).toMatchSnapshot();
});
