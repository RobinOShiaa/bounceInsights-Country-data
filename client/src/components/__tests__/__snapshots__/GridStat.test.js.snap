// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches GridStat component snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="bg-gray-900 px-4 py-6 sm:px-6 lg:px-8"
      >
        <p
          class="text-sm font-medium leading-6 text-gray-400"
        >
          testProperty
        </p>
        <p
          class="mt-2 flex items-baseline gap-x-2"
        >
          <span
            class="text-4xl font-semibold tracking-tight text-white"
          >
            testValue
          </span>
        </p>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="bg-gray-900 px-4 py-6 sm:px-6 lg:px-8"
    >
      <p
        class="text-sm font-medium leading-6 text-gray-400"
      >
        testProperty
      </p>
      <p
        class="mt-2 flex items-baseline gap-x-2"
      >
        <span
          class="text-4xl font-semibold tracking-tight text-white"
        >
          testValue
        </span>
      </p>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
